managed/devops/bin/yb_platform_util.py

```diff
@@ -0,0 +1,715 @@
+# #!/usr/bin/env python
+# Copyright (c) YugaByte, Inc.
+#
+# Copyright 2021 YugaByte, Inc. and Contributors
+#
+# Licensed under the Polyform Free Trial License 1.0.0 (the "License"); you
+# may not use this file except in compliance with the License. You
+# may obtain a copy of the License at
+#
+# https://github.com/YugaByte/yugabyte-db/blob/master/licenses/POLYFORM-FREE-TRIAL-LICENSE-1.0.0.txt
+
+import enum
+import json
+import os
+import copy
+import collections
+import sys
+import argparse
+import textwrap
+import time
+from argparse import RawDescriptionHelpFormatter
+
```

Comment:
Nitpick: may you run `pylint` for the script?

Reply:
jaahir@USBLRJAAHIR1 bin % pylint yb_platform_util.py
************* Module yb_platform_util
yb_platform_util.py:452:9: W0511: TODO need to update API URL once (fixme)
yb_platform_util.py:27:4: E0401: Unable to import 'urllib2' (import-error)
yb_platform_util.py:28:4: E0401: Unable to import 'urllib2' (import-error)
yb_platform_util.py:76:15: W0703: Catching too general exception Exception (broad-except)
yb_platform_util.py:91:28: E0602: Undefined variable 'basestring' (undefined-variable)
yb_platform_util.py:118:15: E0602: Undefined variable 'raw_input' (undefined-variable)
yb_platform_util.py:228:8: R1702: Too many nested blocks (6/5) (too-many-nested-blocks)

------------------------------------------------------------------
Your code has been rated at 9.29/10 (previous run: 9.23/10, +0.06)

jaahir@USBLRJAAHIR1 bin %

Reply:
Last one I am checking. All others are mainly because we have python2 and 3 together. 1st need to wait to merge one PR. If this PR gets merge first then I will pick these changes on that PR.